### PR TITLE
modify commit phase message; improve consensus log info; modify newview logic

### PR DESCRIPTION
**Pull request by chaosma**
modify commit phase message;
improve consensus log information;
modify newview logic;

### CODE REVIEW

**Comment by chaosma** [comment]
Jenkins, this this please.

**Comment by chaosma** [comment]
Jenkins, test this please.

**Comment by chaosma** [comment]
Jenkins, test this please

File: consensus/consensus_v2.go [hunk 1]
```diff
@@ -268,23 +270,23 @@ func (consensus *Consensus) onPrepare(msg *msg_pb.Message) {
...
-		utils.GetLogInstance().Debug("Already received prepare message from the validator", "validatorAddress", validatorAddress)
+		utils.GetLogger().Debug("Already received prepare message from the validator", "validatorAddress", validatorAddress)
 		return
 	}
 
 	// Check BLS signature for the multi-sig
 	var sign bls.Sign
 	err = sign.Deserialize(prepareSig)
 	if err != nil {
-		utils.GetLogInstance().Error("Failed to deserialize bls signature", "validatorAddress", validatorAddress)
```
Comment (mikedoan) [inline consensus/consensus_v2.go]: this morning i was planning to replace all GetLogInstance by GetLogger like this. After talking to EK, i learn that it's not recommended to use GetLogger extensively.

https://github.com/harmony-one/harmony/pull/975 is my closed PR of doing that. Maybe redo it. @astralblue

**Pull request merged**